k8s/chart/templates/configmap-operator.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "sdp-operator.config" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+  {{- include "sdp-operator.labels" . | nindent 4 }}
+data:
+  appgate-operator-host: {{ required "A valid .Values.sdp.operator.host entry required!" .Values.sdp.operator.host }}
+  appgate-operator-device-id: {{ required "A valid .Values.sdp.operator.deviceId entry is required!" .Values.sdp.operator.deviceId }}
+  appgate-operator-log-level: {{ .Values.sdp.operator.logLevel }}
+  appgate-operator-timeout: {{ .Values.sdp.operator.timeout }}
+  appgate-operator-target-tags: {{ .Values.sdp.operator.targetTags }}
+  appgate-operator-exclude-tags: {{ .Values.sdp.operator.excludeTags }}
+  appgate-operator-builtin-tags: {{ .Values.sdp.operator.builtinTags }}
+  appgate-operator-dry-run: {{ .Values.sdp.operator.dryRun }}
+  appgate-operator-cleanup: {{ .Values.sdp.operator.cleanup }}
+  appgate-operator-two-way-sync: {{ .Values.sdp.operator.twoWaySync }}
+  appgate-operator-ssl-no-verify: {{ .Values.sdp.operator.sslNoVerify }}
+  {{ with .Values.sdp.operator.caCert  }}
+  appgate-operator-cacert: {{ . }}
+  {{ end }}
+  {{ with .Values.sdp.operator.fernetKey }}
+  appgate-operator-fernet-key: {{ . }}
+  {{ end }}
+  {{ if .Values.sdp.operator.configMapMt }}
+  appgate-operator-config-map: {{ .Values.sdp.operator.configMapMt }}
+  {{ else }}
+  appgate-operator-config-map: {{- include "sdp-operator.config-mt" . }}
+  {{ end }}
```

Comment:
Is there a reason why we need to pass these values into a configmap then mounted as environment variables in the container? We could directly pass these values into the container's environment and keep the YAML concise.

See conversation in deployment.yaml

Reply:
Ahh I see what you mean. Yeah ,not need for one extra step of indirection. I just reused the way we were deploying it before, mounting the config from a configmap.